packages/discovery-provider/plugins/pedalboard/apps/trending-challenge-rewards/src/trending.ts

```diff
@@ -21,13 +21,8 @@ export const announceTopFiveTrending = async (
   app: App<SharedData>,
   maybeWeek?: string
 ) => {
-  const identityDbRes = await intoResult(async () => app.getIdDb())
-  if (identityDbRes.err) {
-    console.error('Identity connection error: ', identityDbRes)
-    return
-  }
-  const identityDb = identityDbRes.unwrap()
-  const discoveryDb = app.getDnDb()
+  const identityDb = initializeIdentityDb()
```

Comment:
just so I understand, the major diff here is that we're recreating the knex connection once per run versus as a singleton for the life of the whole rewards calculator? I suppose by destroying, we're not leaking any connections, but is this necessary?

Reply:
I could create these as singletons actually, nice callout